source/fe/fe_face.cc

```diff
@@ -645,9 +608,13 @@ FE_FaceQ<1, spacedim>::hp_quad_dof_identities(
 
 template <int spacedim>
 FiniteElementDomination::Domination
-FE_FaceQ<1, spacedim>::compare_for_face_domination(
-  const FiniteElement<1, spacedim> & /*fe_other*/) const
+FE_FaceQ<1, spacedim>::compare_for_domination(
+  const FiniteElement<1, spacedim> & /*fe_other*/,
+  const unsigned int codim) const
 {
+  Assert(codim <= 1, ExcImpossibleInDim(1));
+  (void)codim;
+
   return FiniteElementDomination::no_requirements;
```

Comment:
I am not sure. I just took over what the previous function offered.

Deleting the function would lead to call `compare_for_domination()` of the `FE_PolyFace` since it's its closest parent. We would have to you a distinct assertion here. I'll try that out and run the testsuite.

Reply:
Testsuite does not complain. So I'll add an `Assert(false)` in here.